pkg/cmd/oauth-server/cmd.go

```diff
@@ -19,24 +20,27 @@ import (
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 )
 
-type OsinServer struct {
+type OsinServerOptions struct {
 	ConfigFile string
+	Audit      *options.AuditOptions
 }
 
-func NewOsinServer(out, errout io.Writer, stopCh <-chan struct{}) *cobra.Command {
-	options := &OsinServer{}
+func NewOsinServerCommand(out, errout io.Writer, stopCh <-chan struct{}) *cobra.Command {
```

Comment:
I changed the name to emphasize that it is creating a Cobra Command like it is done in the oauth-apiserver.